tests/test_models.py

```diff
@@ -0,0 +1,55 @@
+from django.test import TestCase
+from taxi.models import Manufacturer, Driver, Car
```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports)